modules/videopress/videopress.php

```diff
@@ -731,6 +759,33 @@ function videopress_shortcode_options( $options ) {
 		return $options;
 	}
 
+	/**
+	 * Get the upload api path.
+	 *
+	 * @param int $blog_id The id of the blog we're uploading to.
+	 * @return string
+	 */
+	protected function make_media_upload_path( $blog_id ) {
+		return sprintf(
+			'%s://%s/rest/v%s/videos/%s/new',
+			'https',
+			JETPACK__WPCOM_JSON_API_HOST,
+			Jetpack_Client::WPCOM_JSON_API_VERSION,
```

Comment:
We may want to hardcode this here -- just because the wpcom rest api version gets a bump doesn't necessarily mean that there's a new version of the endpoint, does it?